doc/source/getting-started/add-license-headers/faq.rst

```diff
@@ -0,0 +1,158 @@
+
+    This would run the hook on python and rst files in the ``src``, ``examples``, ``tests``, and
+    ``doc`` directories, as well as proto and cpp files in any directory.
+
+    .. note::
+
+        The default regex for the ``files`` is ``'(src|examples|tests)/.*\.(py)|\.(proto)'``. Please add onto
+        this regex to specify additional files and directories to ensure the hook runs on python
+        files in the ``src``, ``examples``, and ``tests`` directories, as well as proto files in any
+        directory at the minimum.
+
+.. dropdown:: How do you ignore specific files or file types?
+   :animate: fade-in-slide-down
+
+   To ignore specific files or file types, add the ``exclude`` argument to the hook in your
+   ``.pre-commit-config.yaml`` file:
+
+   .. code:: yaml
+
+      - repo: https://github.com/ansys/pre-commit-hooks
+        rev: v0.5.1
+        hooks:
+        - id: add-license-headers
+          exclude: |
+              (?x)^(
+                  path/to/file1.py |
+                  path/to/.*\.(ts|cpp) |
+                  (.folder1|folder2)/.* |
+                  .*\.js |
+                  \..* |
+              )$
+
+   * ``path/to/file1.py`` excludes the stated file.
+   * ``path/to/.*\.(ts|cpp)`` excludes all .ts and .cpp files within the ``path/to`` directory.
+   * ``(.folder1|folder2)/.*`` excludes directories named .folder1 and folder2.
+   * ``.*\.js`` excludes all .js files in all directories.
+   * ``\..*`` excludes all hidden files.
+
+.. dropdown:: How do you change the copyright phrase from "ANSYS, Inc. and/or its affiliates."?
+    :animate: fade-in-slide-down
+
+    To change the copyright phrase in the copyright line, add the ``--custom_copyright`` argument
+    to the hook in your ``.pre-commit-config.yaml`` file:
+
+    .. code:: yaml
+
+        - repo: https://github.com/ansys/pre-commit-hooks
+          rev: v0.5.1
+          hooks:
+          - id: add-license-headers
+            args:
+            - --custom_copyright="custom copyright phrase"
+
+    This would change the copyright line to ``Copyright (C) 2025 custom copyright phrase.``
+
+.. dropdown:: How do you ignore checking for licensing information in the files?
+    :animate: fade-in-slide-down
+
+    To ignore checking for the MIT license in the files, add the ``--ignore_license_check``
+    argument to the hook in your ``.pre-commit-config.yaml`` file:
+
+    .. code:: yaml
+
+        - repo: https://github.com/ansys/pre-commit-hooks
+          rev: v0.5.1
+          hooks:
+          - id: add-license-headers
+            args:
+            - --ignore_license_check
+
+.. dropdown:: How do you use a custom template?
+    :animate: fade-in-slide-down
+
+    To use a custom template, create the ``.reuse/templates/`` directory in the root of your
+    repository and add the jinja template to that directory. The custom template cannot be named
+    ``ansys.jinja2``, otherwise it will be removed after the hook is done running.
```

Comment:
```suggestion
    repository and add the Jinja template to this directory. The custom template cannot be named
    ``ansys.jinja2``. Otherwise, it would be removed after the hook is done running.
```
Jinja documentation capitalizes Jinja, so we should to. Avoid future tense.